### ISSUE TITLE
Missing text output in annolite(emma_ch1) vignette example

### ISSUE THREAD

**Issue by ChristophLeonhardt** [PolMine/annolite#10]
In annolite version 0.0.1, I tried to run the vignette chunk by chunk. When I run the chunk with the following code:

```
annolite(emma_ch1)
```

instead of the text an empty white box appears within R Studio (below the chunk itself). When I right-click on this space, choose "Inspect Element" and have a look at the console, the first error it reads is 

> Uncaught ReferenceError: crosstalk is not defined

When I knit the entire vignette, the display works, however. 

I already reinstalled crosstalk from CRAN (version 1.1.0.1) but to no avail.

#### Edit:

Maybe interesting: The same behavior occurs in the following chunk: 

```
annolite(
  secretary_general_2000_speech,
  annotations = secretary_general_2000_annotations
)
```

but 

```
emma[["style"]] <- "display:none"
emma_sd <- crosstalk::SharedData$new(emma, ~name, group = "emma") 
crosstalk::bscols(
  widths = c(3,9),
  crosstalk::filter_select(
    id = "select_text", label = "Selection",
    sharedData = emma_sd, group = ~name, multiple = FALSE
  ),
  annolite(emma_sd, annotations = annotationstable())
)
```

works like intended.

**Comment by ablaette** [comment]
Thanks for letting me now. I think the culprit is that the crosstalk dependency is added conditionally to JavaScript, but called irrespective whether crosstalk is used. Will be easy to fix.

**Comment by ablaette** [comment]
Good point: I tried to be parsimonous with the JavaScript dependencies and added the JavaScript part of crosstalk as an dependency in `createWidget()` conditionally, i.e. it was only added for `crosstalk = TRUE`.  However, the check on part of the JavaScript I have written was not robust at all. Fixed, the check I use now is:

```js
if (typeof crosstalk != "undefined"){}
```

This works if crosstalk is undefined.
